requirements-dev.txt

```diff
@@ -10,6 +10,7 @@ flake8-quotes
 geographiclib
 geopandas
 gpxpy
+h5py
```

Comment:
I'm not sure this will fix things. I'm looking into the issue on the coda-forge side. Hold on...

Reply:
BTW, folium does not depend on `h5py` at all. The error is coming from `fiona` b/c of it links to `hdf5`. Please remove that line and wait a little bit as I fix the package upstream.

Reply:
Will revert commit.